ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -29,9 +29,18 @@
   ([icon label]
    [labeled-icon {} icon label])
   ([wrapper-attrs icon label]
-   [:div wrapper-attrs
-    icon
-    [:div (stylefy/use-style style/change-icon-value)
+   [:div (stylefy/use-style
+           (merge {:margin-top "0.5rem"
```

Comment:
You might want to consider moving styles to style file.

Reply:
Done in following PR